sbt_common/messaging/src/test/scala/uk/ac/wellcome/messaging/sqs/SQSWorkerTest.scala

```diff
@@ -28,7 +29,7 @@ class SQSWorkerTest
     with ExtendedPatience
     with fixtures.Akka
     with SQS
-    with MetricsSenderFixture {
+    with test.fixtures.MetricsSender {
```

Comment:
I mildly prefer importing it under a name, as the “test” namespace is quite commonly used!